docs/satellite-clone.md

```diff
@@ -91,3 +91,25 @@ On the target server:
       # ansible-playbook satellite-clone-playbook.yml
     ```
   The playbook run may may take a while to complete.
+
+## Restoring a Satellite With a Remote Database
+
+Note: the backup used for this procedures must have the correct options to manage remote databases in `/etc/foreman-installer/scenarios.d/satellite-answers.yaml`. Please check that the answers file has the correct database management values filled.
```

Comment:
probably should actually check this in the playbook, the `manage-db` type options not matching in the backup and the target satellite will cause issues. For instance, if I have a backup that doesn't have foreman-manage-db set to true, but I pass in that option to clone, it will install foreman remotely, but then `foreman-maintain` will restore the backup, see that foreman-manage-db is false, and try to set up the database locally. This is because foreman-maintain uses `satellite-installer --reset`